### ISSUE TITLE
Feature request: element-types should also check export

### ISSUE THREAD

**Issue by iliubinskii** [javierbrea/eslint-plugin-boundaries#213]
**Is your feature request related to a problem? Please describe.**

Let's assume I want to disallow importing b.js:
```ts
a.js:
import * as b from "b.js"; // Warning
```

But:
```ts
c.js:
export * from "b.js"; // No warning, because export is not checked

a.js:
import * as b from "c.js"; // No warning, because I disallowed b.js, but not c.js
```

**Describe the solution you'd like**
element-types should also check export

**Comment by orcwarrior** [comment]
Hi! The whole idea of the plugin is brilliant! But yeah, I'm looking for the same functionality and even more - I would like my config to be fine-grained, so I could specifically define possible import and export patterns separately. Started with forking this repo, will see what comes next

**Comment by javierbrea** [comment]
> Hi! The whole idea of the plugin is brilliant! But yeah, I'm looking for the same functionality and even more - I would like my config to be fine-grained, so I could specifically define possible import and export patterns separately. Started with forking this repo, will see what comes next

Thanks @orcwarrior, I'm glad you like the idea! Please let me know about your advances, maybe we can add them to the plugin. Or feel free to contribute to the project opening a PR! 😃

**Comment by javierbrea** [comment]
This can be made by using the next setting, available from the version [4.0.0](https://github.com/javierbrea/eslint-plugin-boundaries/releases/tag/v4.0.0):

```jsonc
{
  "settings": {
    "boundaries/dependency-nodes": ["import", "export"]
  }
}

```
